h2o-automl/src/main/java/ai/h2o/automl/SortMetric.java

```diff
@@ -0,0 +1,12 @@
+package ai.h2o.automl;
+
+public enum SortMetric {
```

Comment:
the main reason I'm hesitating is because it partly duplicates `StoppingMetric`, without the useful logic in the latter that could be used to remove duplicate logic in `Leaderboard`, and without the potential support for custom metric. Also, it would need AUTO.
I would prefer having this new enum used only in `AutoMLV99` and have it immediately converted to `StoppingMetric` and then have all the logic using only `StoppingMetric`

Reply:
Sounds good!, Big fan of introducing `AUTO`. 

@seb-h2o Should I leave this PR open for you to work on it? I would like to spend on more urgent features in Sparkling Water

Reply:
I just wanted to try and see if 5-minute solution would do, but the issue is more complex. Would rather let AutoML dev to deal with it

Reply:
sure, please leave it, I'm filing a Jira.
Thx @jakubhava